### PR TITLE
Add and replaced text to the "keep it simple" part of the introduction

### PR DESCRIPTION
**Pull request by earlgray1979**
Elaborated some of the text and slightly changed the style, as well as correcting minor mistakes.

### CODE REVIEW

File: documentation/get-started/concepts.en.md [hunk 1]
```diff
@@ -9,18 +9,11 @@ QField was designed with a few key concepts in mind.
...
-not.
-
-This means that everything like layer styling, form definitions and
-other project setup steps should be done on a computer with QGIS
-installed first.
+The requirements in the field are hardly the same as on a desktop. Firstly, using a phone or a tablet, the canvas to work on is much more limited than on a computer monitor. Secondly, devices for data collection as well as the individual tasks that need to be performed are vastly different.
+
+QField aims to help users do fieldwork without the nuisance of a cluttered user interface. In other words, only the relevant parts of a task are accessible on the interface, while less important things remain out of sight.
+
+Before heading out to the field though, preliminary steps such as layer styling, tailored forms, or in general any other project configurations should first be done on the QGIS desktop version.
```
Comment (suricactus) [inline documentation/get-started/concepts.en.md:16]:
```suggestion
The requirements in the field are hardly the same as on a desktop.
Firstly, using a phone or a tablet, the canvas to work on is much more limited than on a computer monitor.
Secondly, devices for data collection as well as the individual tasks that need to be performed are vastly different.

QField aims to help users do fieldwork without the nuisance of a cluttered user interface.
In other words, only the relevant parts of a task are accessible on the interface, while less important things remain out of sight.

Before heading out to the field though, preliminary steps such as layer styling, tailored forms, or in general any other project configurations should first be done on the QGIS desktop version.
```

**Review by suricactus — COMMENTED** [review]
Hey, thanks for your contributions!

Since you are already working on the documentation blocks, it would be great if you target having a single sentence per line. If you don't put a new empty line between the sentences, markdown will treat them as part of one paragraph.

The benefit of such change is when one has to add or rephrase a sentence, it does not affect multiple rows. Also somehow enforces shorter sentences.

File: documentation/get-started/concepts.en.md (hunk 1 above)
Comment (earlgray1979) [reply documentation/get-started/concepts.en.md:16]:
```suggestion
The requirements in the field are hardly the same as on a desktop. Firstly, using a phone or a tablet, the canvas to work on is much more limited than on a computer monitor. Secondly, devices for data collection as well as the individual tasks that need to be performed are vastly different.

QField aims to help users do fieldwork without the nuisance of a cluttered user interface. In other words, only the relevant parts of a task are accessible on the interface, while less important things remain out of sight.

Before heading out to the field though, preliminary steps such as layer styling, tailored forms, or in general any other project configurations should first be done on the QGIS desktop version.
```

Comment (earlgray1979) [reply documentation/get-started/concepts.en.md:16]: Thanks alot for the changes, I'll create a new "Concepts" - branch and incorporate the changes.

**Comment by suricactus** [comment]
Closing this as the changes are integrated in #471
